components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt

```diff
@@ -38,6 +41,21 @@ class ExperimentsDebugActivity : Activity() {
             }
         }
 
+        if (intent.hasExtra(SET_KINTO_INSTANCE_EXTRA_KEY)) {
+            val instance = intent.getStringExtra(SET_KINTO_INSTANCE_EXTRA_KEY)
+
+            val config: Configuration? = when (instance) {
+                "dev" -> Configuration(kintoEndpoint = ExperimentsUpdater.KINTO_ENDPOINT_DEV)
```

Comment:
Note that `Configuration()` will drop any custom value that was previously provided as part of the configuration (e.g. the http client), and fall back to the defaults. What you really want here is to do `Configuration.copy(kintoEndpoint = ...)`

Reply:
Thanks, I will fix this as I get back to work on this PR!